docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -200,20 +205,50 @@ $ tsh clusters
 # example.com online root                *        
 ```
 
-## Configuring hardware key support with per-session MFA
+## Custom PIV setup
+
+### Custom PIV Slot
+
+By default, Teleport clients use the following PIV slots for each option:
+  - `hardware_key`: slot `9a`
+  - `hardware_key_touch`: slot `9c`
+  - `hardware_key_pin`: slot `9d`
+  - `hardware_key_and_pin`: slot `9e`
+
+If you are using any other PIV applications, you may need to specify a different slot to be used.
+For example, `yubikey-agent` uses slot `9a`, so a user with the `hardware_key` requirement would need
+to specify a different slot to avoid overwriting their `yubikey-agent` key and cert.
```

Comment:
```suggestion
For example, `yubikey-agent` uses slot `9a`. To avoid overwriting the `yubikey-agent` key and certificate, a user with the `hardware_key` requirement would need to specify a different slot.
```